src/javascript/note.js

```diff
@@ -13,9 +13,11 @@ export default class Note {
     this.isHoldNote = keyObj.h && keyObj.h[key];
     this.isUserHolding = false;
     this.isHoldingDone = false;
+    this.didUserHold = false;
 
-    this.y = 0;
+    this.y = y ?? 0;
```

Comment:
```suggestion
    this.y = y;
```

Let's favour default arguments here then?

Reply:
Yep

Reply:
Will need to reconsider later